.github/workflows/validate-release.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Install JQ
         run: brew install jq
       - name: Validate the release
-        run: VERSION=${{ steps.get_version.ouputs.VERSION }} ./release-scripts/validate-osx-release.sh
```

Comment:
not sure if this would work anyway -- I think this needs to be inside of a separate `env` key